src/coupled_driver.cpp

```diff
@@ -115,6 +116,24 @@ void CoupledDriver::compute_temperature_norm(const CoupledDriver::Norm& n,
   converged = norm < epsilon_;
 }
 
+bool CoupledDriver::is_converged()
+{
+  bool converged;
+
+  // assumes that the rank 0 process is in the communicator that has access
+  // to global coupling data
+  if (comm_.rank == 0) {
+    double norm;
+    compute_temperature_norm(Norm::LINF, norm, converged);
+
+    std::string msg = "temperature norm_linf: " + std::to_string(norm);
+    comm_.message(msg);
+  }
+
+  err_chk(comm_.Bcast(&converged, 1, MPI_CXX_BOOL));
```

Comment:
I'm not sure how this `err_chk` got here originally, but it shouldn't be there. `err_chk` was specifically for checking the error on OpenMC C API calls.

Reply:
If you get rid of it, you can also get rid of the error.h include above.